pkg/locker/locker_api.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package locker
+
+// package locker provides security API for protecting data for storage and/or transfer
+
+// Locker is a service that protects a message
+type Locker interface {
+	// Lock a message msg into ciphertext
+	Lock(msg []byte) ([]byte, error)
+	// Unlock a ciphertext and return the locked message
```

Comment:
similar to above, description is not clear - isn't it about returning the unlocked key?

Reply:
The locker can encrypt and decrypt any message - including keys.

Reply:
But I thought that we are introducing Locker to work with a narrower scope, that is - keys. Does it makes sense to "lock" a message - there is already an operation for that, and it's Encrypt.

Reply:
Well... good point. Maybe we don't need a new abstraction called `Locker`.

Reply:
Indeed! I was thinking about this all night after pushing the PR and noticed the locker service already exists in the framework. The novelty here is secret splitting. I'll just update the patch to expose  functions and remove the locker service.

Reply:
@Baha-sk part of the point here is... whatever we are calling a "locker" here is really nothing more than a "crypter".

Reply:
yes @llorllale .. we don't need an extra service for this